tests/PHPStan/Rules/Exceptions/CaughtExceptionExistenceRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Exceptions;
+
+class CaughtExceptionExistenceRuleTest extends \PHPStan\Rules\AbstractRuleTest
+{
+
+	protected function getRule(): \PHPStan\Rules\Rule
```

Comment:
@bogdan-rizac Disagree. The native typehint already contains everything there's to be said about this method. It's also part of our coding standard - remove all phpDocs that do not add any value.